src/test/routes/search.ts

```diff
@@ -14,25 +14,45 @@ describe('search', () => {
     test('should redirect search location page', async () => {
       await request(app)
         .post('/search-option')
-        .send({ knowName: 'yes' })
+        .send({ knowLocation: 'yes' })
         .expect((res) => expect(res.status).to.equal(302));
     });
 
     // TODO this will change in the future when we have other stories implemented
     test('should redirect home page', async () => {
       await request(app)
         .post('/search-option')
-        .send({ knowName: 'no' })
+        .send({ knowLocation: 'no' })
         .expect((res) => {
           expect(res.status).to.equal(302);
         });
     });
+
+    test('should render the search option if no options was selected', async () => {
+      await request(app)
+        .post('/search-option')
+        .expect((res) => {
+          expect(res.status).to.equal(200);
```

Comment:
the test doesn't do what it says it does

Reply:
theses have been removed as discuess, routes just point to the controller so we can only really check the status